upload/library/SV/WarningImprovements/Installer.php

```diff
@@ -84,6 +84,14 @@ public static function install($existingAddOn, $addOnData)
             ));
         }
 
+        if ($version < 1040100) {
```

Comment:
This SQL needs to be executed before the previous block; otherwise you get the following error on upgrading from xf1.3.x;
```
Mysqli prepare error: Unknown column 'allowed_user_group_ids' in 'field list'
```